src/components/Main.js

```diff
@@ -22,15 +22,21 @@ class Main extends React.Component{
   
   constructor(props){
       super(props);
-      //this.state =;
+      
     this.addTodo = this.addTodo.bind(this);
+    this.state = {value:''};
+
+  }
+  textboxChangeHandler = (e) =>{
+    e.persist();
+    this.setState({value : e.target.value});
 
   }
   addTodo = () => {
-     console.log('in todo');   
-    const todo = { task: document.getElementById("textbox1").value, status :'pending'}
-    
-      this.props.addTodo(todo);
+     
+    const todo = { task: this.state.value, status :'pending'}
```

Comment:
you can make a global variable and define states over there as `PENDING`, `COMPLETED`, etc. instead of manually adding string.

Reply:
Noted. Will add changes.